### PR TITLE
Development setup in readme, and some adjustments

### PR DESCRIPTION
**Pull request by jancborchardt**
As mentioned in https://github.com/nextcloud/server/pull/12186#issuecomment-435017943, adding some info regarding frontend development setup to the main readme.

Also fix some structure of readme, adjust wording, and add more emojis 🤩

Please review @skjnldsv @juliushaertl @camilasan @rullzer @danxuliu @MariusBluem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,43 +9,100 @@
...
 ## Join the team :family:
 
-### How to contribute
+### How to contribute 👩‍💻
 
 1. [Set up your local development environment](https://docs.nextcloud.com/server/14/developer_manual/general/devenv.html) :rocket:
 2. [Pick a good first issue](https://github.com/nextcloud/server/labels/good%20first%20issue) :notebook:
-3. Create a branch, a [Pull Request](https://opensource.guide/how-to-contribute/#opening-a-pull-request) and `@mention` the people from the issue :computer:
-4. Wait for it to get merged and :tada:
+3. Create a branch, a [pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request) and `@mention` the people from the issue to review :computer:
```
Comment (juliushaertl) [inline README.md]: Can we add a note about adding the sign off message here? This is not obvious from the readme and missing in a lot of first time contributors PRs.

**Comment by jancborchardt** [comment]
Also added sentence about non-technical contributions with link to https://nextcloud.com/contribute/, and fixed the screenshot! (@jospoortvliet @MariusBluem if the structure in the Screenshots repo is changed, links will break. ;) We should keep the naming.)

File: README.md (hunk 1 above)
Comment (jancborchardt) [reply README.md]: Added! Is it good like that?

**Review by rullzer — APPROVED** [review]
Nice :)

File: README.md [hunk 2]
```diff
@@ -7,45 +7,104 @@
...
+```
+./build/compile-handlebars-templates.sh
+```
+
+
+### Tools we use 🛠
+
+[![BrowserStack](https://user-images.githubusercontent.com/45821/41675934-61fa3442-74c4-11e8-8c8e-90768c56ba08.png)](https://www.browserstack.com/)
+[![WAVE](https://wave.webaim.org/img/wavelogo.png)](https://wave.webaim.org/extension/)
+[![](https://developers.google.com/web/tools/lighthouse/images/lighthouse-icon-128.png) Lighthouse](https://developers.google.com/web/tools/lighthouse/)
```
Comment (splitt3r) [inline README.md]:
```suggestion
[![Lighthouse](https://developers.google.com/web/tools/lighthouse/images/lighthouse-icon-128.png)](https://developers.google.com/web/tools/lighthouse/)
```

**Review by splitt3r — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (splitt3r) [reply README.md]: Personally i think there are to many smileys now :grin:. Anyway i think the logos under "Tools we use" should be a bit smaller or even same size.

Comment (jancborchardt) [reply README.md]: @splitt3r I see why you suggest that. :) This is intentional though as the logo I found has no text. And the others do have text in the logo. So for Lighthouse it is indeed text, and not just alternative text for the image.

**Review by skjnldsv — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jancborchardt) [reply README.md]: > Anyway i think the logos under "Tools we use" should be a bit smaller or even same size.

Good point! Switched to using HTML in that section and reduced the size.

Comment (jancborchardt) [reply README.md]: Actually didn’t really work out with the HTML – just switched to text-only. :)

Comment (juliushaertl) [reply README.md]: I'm not sure if browserstack required to include the logo for the free accounts, but @rullzer probably knows.

**Pull request merged**